app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java

```diff
@@ -250,6 +287,8 @@ private void init(@Nullable AttributeSet attrs) {
         resizeDrawable1.setTint(cornerColor);
         resizeDrawable2.setTint(cornerColor);
         resizeDrawable3.setTint(cornerColor);
+        // keep aspect ratio
```

Comment:
the variable says the function, Is it necessary the comment?

Reply:
Ok